pkg/artifactory/provider.go

```diff
@@ -63,8 +63,8 @@ func Provider() *schema.Provider {
 			"check_license": {
 				Type:        schema.TypeBool,
 				Optional:    true,
-				Default:     true,
-				Description: "Toggle for pre-flight checking of Artifactory license. Default to `true`.",
+				Default:     false,
```

Comment:
why change the default? No one is ever going to set this to true

Reply:
it could create more problems than solve, and users won't know about their license problem right away

Reply:
Some of the calls don't need the license. So, yeah, the default should be false, otherwise, a lot of users affected.

Reply:
Also, not all resources require admin access which this license API requires.

Reply:
Well, then let's just ditch it and allow for late failure - if it's disabled by default, then by default it's late failure.